articles/service-fabric/how-to-managed-cluster-availability-zones.md

```diff
@@ -218,6 +218,32 @@ Requirements:
    ```
    If you run in to any problems reach out to support for assistance.
 
+## Enable FastZonalUpdate on Service Fabric managed clusters (preview)
+Service Fabric managed clusters support faster cluster and application upgrades by reducing the max upgrade domains per availability zone.   
```

Comment:
We need to be more descriptive about the configuration. 
Do add some background aswell: The default configuration right now can have atmost 15 UDs in the multiple nodetype. This huge number of UDs reduced the upgrade velocity. Using the new configuration the max UDs is reduced which results in faster updates, keeping the safety intact the safety of the upgrades.

Reply:
ack done.  and added that each nodetype needs new schema

Reply:
@abhayshah9 can you please rephrase to explicitly mention that this update has to be done for every nodetype. Then talk about how to do it ?

Reply:
We should also mention why is the step 2 & 3 required.

Reply:
got it.  added and updated.